src/app/pages/calendar-page/calendar-page.component.ts

```diff
@@ -48,7 +58,7 @@ export class CalendarPageComponent implements OnInit, OnDestroy {
     },
     customButtons: {
       submit: {
-        text: 'Submit',
+        text: `Submit`,
```

Comment:
why those have changed?
Do we need ` here?

Reply:
It was changed to a dynamic value, but then returned. forgot to change